### DISCUSSION TITLE
[Question]: Frigate Sublabels

### DISCUSSION THREAD

**Discussion by Ostedtinkerer** [blakeblackshear/frigate#11982] [Ask A Question] [answered]
### What is your question:

Hi!

I went from a successfully V13 with working sublabels instance to V14. - Everything is working great, but not Sublabels. From "frigate_plate_recognizer".  https://github.com/ljmerza/frigate_plate_recognizer
The license plate server works without problems and read the frames properly from frigate, but frigate V14 doesn't show Sublabels.
- Will this be available in the future?

**Comment by NickM-27** [comment]
where are you looking for the sub labels and not seeing them?

**Comment by Ostedtinkerer** [comment]
It could be showed in the Filters menu in V13.
- can it be showed else where?
Then under filters, I could filter out the specific plate by pictures, name and plate registration.

I can see the license plate server is working correctly according to logs. - It is returning a sub label.

Sendt fra Outlook til Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by NickM-27** [reply, marked as answer]
in 0.14 sub labels are shown when you hover on the chip in the top left corner of a review item. They are currently not filterable

**Reply by Ostedtinkerer** [reply]
I'll take a look. Thank you for response.
